unit-tests-jest/numbers.test.ts

```diff
@@ -0,0 +1,49 @@
+// import { isExportDeclaration } from 'typescript';
+import { evenNumbers, toDollars, multiplyBy, divideBy } from './numbers';
+describe('return even nums', () => {
```

Comment:
Usually you would put the name of the function you are testing here:
`describe('evenNumbers', () => {`